spa/webpack/webpack.dev.ts

```diff
@@ -1,25 +1,34 @@
+// import fs from 'fs';
 import path from 'path';
 import webpack from 'webpack';
 import {merge} from 'webpack-merge';
 import common from './webpack.common.js';
+import webpackDevServer from 'webpack-dev-server';
 
 const dirname = process.cwd();
-const devConfig: webpack.Configuration = {
+const devServer: webpackDevServer.Configuration = {
 
-  mode: 'development',
-  devServer: {
+    /*server: {
```

Comment:
I tweaked this a little for a use case to show the updated dev server settings if you [adapt the deployment to use SSL](https://curity.io/resources/learn/token-handler-deployment-example/#adapt-the-deployment)